app/controllers/resources_controller.rb

```diff
@@ -158,6 +175,10 @@ def blob_for_signed_id(signed_id, filename)
     raise BlobError, "Unable to find upload for #{filename} (#{signed_id})"
   end
 
+  def file_from_globus(signed_id, filename)
```

Comment:
I tend to think of `signed_id` as an ActiveStorage-ism. Do we have a signed ID here or just a globus URI? I wonder about changing some of the variable names for extra clarity down the road.